tests/integration/static_ip_address_test.py

```diff
@@ -58,6 +58,26 @@ def setup_eth1_ipv4():
     netapplier.apply(desired_state)
 
 
+@pytest.fixture
+def setup_eth1_ipv6():
```

Comment:
Multiple tests are using the fixture, therefore, it is better add a teardown to remove the address.
On the other hand, we have a general fixture (in conftest) which takes eth1/2 down and up again before each test.

Ahhh that `cleanup_test_interfaces` needs to be changed, it is a bad too wide fixture.